### PR TITLE
Merge from scraping-update development branch

### PR DESCRIPTION
**Pull request by qu-gg**
- Add new GunTemplate
- Fix scraping to get higher quality images
- Update gun generation parameters for better output
- Regenerated scraped files
- Add new output example

### CODE REVIEW

**Comment by qu-gg** [comment]
Going to request that you try and run this scraper locally on your end to see if you can get the full webpage for all the gun arts or if there is something I broke in the change.

Unsure why it's only picking up a fraction (71 samples) of the gun lists - tried different html protocols.

File: local_app.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+from classes.Gun import Gun
+from classes.GunImage import GunImage
+from generate_gun_cmd import generate_gun_pdf
+from classes.json_reader import get_file_data
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtGui import QIcon
+from PyQt5 import QAxContainer
+from PyQt5.QtWidgets import (QApplication, QComboBox, QGridLayout, QGroupBox, QLabel,
+                             QLineEdit, QWidget, QPushButton, QCheckBox,QMainWindow)
```
Comment (Milkshak3s) [inline local_app.py:21]: I thought we were django this time F

**Review by Milkshak3s — APPROVED** [review]

File: local_app.py (hunk 1 above)
Comment (qu-gg) [reply local_app.py:21]: WebApp yes, local app no

**Review by Milkshak3s — APPROVED** [review]
🚀

**Pull request merged**